internal/venice-test-common/src/integrationTest/java/com/linkedin/venice/controller/kafka/consumer/AdminConsumptionTaskIntegrationTest.java

```diff
@@ -319,6 +418,30 @@ private byte[] getStoreDeletionMessage(String clusterName, String storeName, lon
     adminMessage.operationType = AdminMessageType.DELETE_STORE.getValue();
     adminMessage.payloadUnion = deleteStore;
     adminMessage.executionId = executionId;
-    return adminOperationSerializer.serialize(adminMessage);
+    return adminOperationSerializer
+        .serialize(adminMessage, AdminOperationSerializer.LATEST_SCHEMA_ID_FOR_ADMIN_OPERATION);
+  }
+
+  private byte[] getStoreUpdateMessage(
+      String clusterName,
+      String storeName,
+      String owner,
+      long executionId,
+      int writerSchemaId) {
+    UpdateStore updateStore = (UpdateStore) AdminMessageType.UPDATE_STORE.getNewInstance();
+    updateStore.clusterName = clusterName;
+    updateStore.storeName = storeName;
+    updateStore.owner = owner;
+    updateStore.partitionNum = 20;
+    updateStore.currentVersion = AdminConsumptionTask.IGNORED_CURRENT_VERSION;
+    updateStore.enableReads = true;
+    updateStore.enableWrites = true;
+    updateStore.replicateAllConfigs = true;
```

Comment:
Could you please leave a TODO message here and see if the tests starting to fail once we introduce new fields semantic checks in milestone 3?

Reply:
Added.